lib/components/LazyPicture.vue

```diff
@@ -1,18 +1,17 @@
 <template>
-  <lazy-image v-bind="lazyImage">
+  <lazy-image v-bind="$attrs" :src="lazyImage.src" :srcset="lazyImage.srcset">
     <!-- eslint-disable-next-line vue/no-template-shadow -->
     <template slot-scope="{ lazy }">
       <picture v-if="lazy">
         <source v-for="(source, index) in preparedSources" :key="index" v-bind="source">
-        <img :src="placeholder" :alt="$attrs.alt" :title="$attrs.title" loading="eager">
+        <img :src="fallback" v-bind="$attrs" loading="eager">
```

Comment:
Sortier die perparedSources noch anhand des Mediaqueries. So kann der User in x-beliebiger Reihenfolge die Sourcen angeben.

Reply:
Nutz hier doch das CustomImage + und übergib mit das SizeObject vom Fallback Image

Reply:
> Sortier die perparedSources noch anhand des Mediaqueries. So kann der User in x-beliebiger Reihenfolge die Sourcen angeben.

Wie nach Media Query sortieren?, diese sind ja doch sehr flexibel.

Reply:
ich dacht an so etwas hier https://github.com/niksy/sort-media-queries